readme.md

```diff
@@ -146,6 +155,13 @@ Execute a command synchronously through the system shell.
 
 Returns the same result object as [`child_process.spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options).
 
+### spawned.cancel()
```

Comment:
It's not clear enough what `spawned` is. And it's weird to place the docs here. I think it would be better to document it in the `execa()` section.